tephra-hbase-compat-0.96/src/main/java/co/cask/tephra/hbase96/coprocessor/TransactionProcessor.java

```diff
@@ -198,6 +209,40 @@ public RegionScanner preScannerOpen(ObserverContext<RegionCoprocessorEnvironment
     return s;
   }
 
+  /**
+   * Ensures that family delete markers are present in the columns requested for any scan operation.
+   * @param scan The original scan request
+   * @return The modified scan request with the family delete qualifiers represented
+   */
+  private Scan projectFamilyDeletes(Scan scan) {
```

Comment:
The idea was that you are "projecting" the family delete qualifer into the results, ala schema projection.  Not a big deal to rename though.